app/controllers/steps/details/start_controller.rb

```diff
@@ -1,7 +1,7 @@
 module Steps::Details
   class StartController < Steps::DetailsStepController
     def show
-      @can_start = current_tribunal_case&.cost_task_completed?
+      @can_start = current_tribunal_case.present?
```

Comment:
This determination and the one in the lateness start controller can probably go altogether.  For now, however, I wanted to get rid of the cost_task reference with a minimal amount of refactoring.  Will put a ticket in to review this as a chore later.